sphinx/texinputs/sphinx.sty

```diff
@@ -1164,26 +1164,19 @@
 % {fulllineitems} is the main environment for object descriptions.
 %
 \newcommand{\py@itemnewline}[1]{%
+  \kern\labelsep
   \@tempdima\linewidth
-  \advance\@tempdima \leftmargin\makebox[\@tempdima][l]{#1}%
+  \advance\@tempdima \labelwidth\makebox[\@tempdima][l]{#1}%
+  \kern-\labelsep
 }
 
-\newenvironment{fulllineitems}{
-  \begin{list}{}{\labelwidth \leftmargin \labelsep \z@
+\newenvironment{fulllineitems}{%
+  \begin{list}{}{\labelwidth \leftmargin
                  \rightmargin \z@ \topsep -\parskip \partopsep \parskip
                  \itemsep -\parsep
-                 \let\makelabel=\py@itemnewline}
+                 \let\makelabel=\py@itemnewline}%
 }{\end{list}}
 
-% Redefine description environment so that it is usable inside fulllineitems.
-%
-% FIXME: use sphinxdescription, do not redefine description environment!
-\renewcommand{\description}{%
-  \list{}{\labelwidth\z@
-          \itemindent-\leftmargin
-	  \labelsep5pt\relax
```

Comment:
the default labelsep is `0.5em` which usually is `5pt` for `10pt` documents. But this usage of `5pt` was not good for documents with `11pt` or `12pt` option.

anyway, this is not needed anymore, hence I removed it. Documents with `11pt` or `12pt` options will have very slight whitespace change in description lists, but this is for the better.